src/main/java/org/cbioportal/session_service/domain/SessionType.java

```diff
@@ -0,0 +1,8 @@
+package org.cbioportal.session_service.domain;
+
+public enum SessionType {
```

Comment:
@kalletlak this might be a good place to describe what each session is? Or maybe the `README`

Reply:
@inodb added description in `README`